ISSUE_TEMPLATE.md

```diff
@@ -14,6 +14,21 @@ Large features typically go through three stages: [Alpha, Beta, and Stable](http
 Each stage requires various approvals from various teams.  Features require several releases
 to reach Stable.
 
+The following items have to be filled before the features freeze:
+
+- Primary contact (assignee);
+- SIG, responsible for the feature;
+- Proposal link - do the public design proposal in the community repo;
+- Reviewer(s) - recommend having 2+ reviewers (at least one from code-area OWNERS file) agreed to review. Reviewers from multiple companies preferred;
+- Approver (for LGTM, likely from SIG to which feature belongs);
+- Target - Alpha / beta / stable;
+- Timeline / status;
+
+Before the coding freeze:
+
+- Release note - a short sentence, describing the feature;
+- User documentation PR link (to http://github.com/kubernetes/kubernetes.github.io)
```

Comment:
I am in favor of pairing feature PRs with doc PRs, but the doc repo does not currently accept PRs for the release under development. Doc PRs to master for current release work cannot be merged until the release is cut, and there is no release branch for PRs until late in the dev cycle (there is still no release-1.6 branch open for doc PRs). 

/cc @jaredbhatti